### DISCUSSION TITLE
[FR] colorized addon icon

### DISCUSSION THREAD

**Discussion by cociweb** [music-assistant/support#1525] [Feature requests and ideas]
Hello,
The MA2 addon icon is very fency and I like it. However, it is black&white. Unfortunatelly, HA converts the icon color of an inactive addon to grey scaled which will be also black and white. Can you please add some color into the icon/logo to clearly distinguish when the on is not running?

Thank's

**Comment by OzGav** [comment]
We will see what the future holds but this will be low priority. If you are using HAOS then turn on the watchdog so the server should always be running.

**Reply by cociweb** [reply]
Sure, it's really just a glitch with a very-very low prio. I know (and I see the backlog), there are more important issues. - It's reasonable.
Anyway, should we convert this discussion to issue to be in the backlog?

**Reply by OzGav** [reply]
I’ll talk to the team about it. If they are happy to look at this in the future I’ll move this across. Thanks.

**Comment by cociweb** [comment]
Just necro-bumping the thread. :) - [off] I really like the 'Advent anticipation' in every year: surprises and packages :)

I've just saw some promising orange colored favicon https://music-assistant.io/assets/favicon-orange.png :)

There isn’t a query on this occasion only an eagle eyed fact-finding (to avoid any potential misunderstanding related to the term ‘hasten’). - You guys are the masters of musicplayers.

**Comment by OzGav** [comment]
This will be implemented in the next beta release.

**Reply by cociweb** [reply]
Great, but I've changed from supervised to the container version of HA (since supervised is deprecated), so it is not relevant for me anymore. :) However, for HAOS users, it can be useful.

**Comment by cociweb** [comment]
Just for reference:
https://github.com/music-assistant/home-assistant-addon/commit/5e727d535ace48187cb02c82ec0d70b87611f518

Thanks @OzGav!
